flow/designs/asap7/mock-array/Element/config.mk

```diff
@@ -43,3 +43,7 @@ export PLACE_PINS_ARGS = -annealing
 
 export GND_NETS_VOLTAGES      =
 export PWR_NETS_VOLTAGES      =
+
+export CTS_ARGS = -insertion_delay -sink_clustering_enable -balance_levels -distance_between_buffers 60
```

Comment:
You appear to have mixed your other PR into this one.  This serializes the merge process.